### PR TITLE
IGAPP-432 adjust documentation for aschaffenburg app

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ versioning: ![versioning](https://img.shields.io/badge/calver-YYYY.MM.PATCH-22bf
 
 # integreat-app
 
-React JS and React Native App for [Integreat](https://integreat-app.de) and [Malte](https://www.malteser-werke.de/malte-app.html).
+React JS and React Native App for [Integreat](https://integreat-app.de), [Malte](https://www.malteser-werke.de/malte-app.html) and [Aschaffenburg](https://aschaffenburg.app)
```
Comment (klinzo) [inline README.md:6]:
```suggestion
React JS and React Native App for [Integreat](https://integreat-app.de), [Malte](https://www.malteser-werke.de/malte-app.html) and [Aschaffenburg](https://aschaffenburg.app).
```
I think a project overview would be better. But as we don't have that :shrug:

File: docs/cicd.md [hunk 2]
```diff
@@ -22,12 +22,12 @@ If no branch is specified, main is used as default. Per default the delivery is
 ## Workflows
 
 Several workflows exist for different purposes:
-* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat and Malte webapps on the main branch.
-* **scheduled_native_development_delivery**: Delivers Integreat and Malte native builds to development (Testflight and Beta channel)
+* **commit**: Run for all commits of PRs to ensure good code quality and working code. Delivers Integreat, Malte and AschaffenburgApp webapps on the main branch.
```
Comment (klinzo) [inline docs/cicd.md:25]: I think we should just stick with `Aschaffenburg`

**Review by klinzo — APPROVED** [review]

File: docs/cicd.md (hunk 2 above)
Comment (Taggotty) [reply docs/cicd.md:25]: Agree

**Review by Taggotty — APPROVED** [review]

File: docs/cicd.md (hunk 2 above)
Comment (SteffiStuffel) [reply docs/cicd.md]: done

**Pull request merged**